source/common/stats/histogram_impl.h

```diff
@@ -81,12 +81,15 @@ class HistogramImpl : public HistogramImplHelper {
   }
 
   // Stats::Histogram
+  Unit unit() const override { return unit_; };
   void recordValue(uint64_t value) override { parent_.deliverHistogramToSinks(*this, value); }
 
   bool used() const override { return true; }
   SymbolTable& symbolTable() override { return parent_.symbolTable(); }
 
 private:
+  Unit unit_;
```

Comment:
FWIW this is likely where the memory-consumption difference comes from, and this makes sense. No response needed on this; you can just adjust the expected value as needed.

Reply:
Yep, the mem alignment changed and it was off both ways actually. Adjusted.